routes.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const app = express();
+const data = require('./data.json');
+
+app.use(express.json());
+
+
+const get = app.get('/get', (req, res) => {
+    res.send(data);
+});
+
+const post = app.post('/post', (req, res) => {
```

Comment:
It's a good practice to validate the incoming request data before processing it. Consider using a library like 'joi' for this purpose. [medium]